components/FixedBanner.vue

```diff
@@ -121,7 +131,12 @@ export default {
       <div class="banner-dialog-glass"></div>
       <div class="banner-dialog">
         <div class="banner-dialog-frame" :style="dialogStyle">
-          <div class="banner" :style="bannerStyle">
+          <div v-if="Array.isArray(banner.text)" class="banner" :style="bannerStyle">
```

Comment:
Can we replace this with a computed value, so we can avoid logic in the markup?

It may be worthy to add some comments to why do we check if is array as well.